### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by keebot-png**
## In this project I had done the following:
- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Followed wireframe design.
- Used React Router.
- Passed linter errors.

### CODE REVIEW

File: src/components/calculatorPage/calculator.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  const { total, next } = state;
+  let result = 0;
+  if (total === null && next !== null) {
+    result = next;
+  } else if (total !== null && next === null) {
+    result = total;
+  } else {
+    result = next;
+  }
+
```
Comment (tufoinnkuo10) [inline src/components/calculatorPage/calculator.js:25]: - [ ] Great job. you can add your style to improve the look and feel but kindly follow the wireframe design. please add the math quotes and align your calculator to the right like in the `wireframe`. 

wireframe
Screenshot (583)_LI


yours 
Screenshot (585)
5c5af61.jpg)

File: src/components/navbarHeading/navbar.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './navbar.css';
+
+const Navbar = () => (
+  <nav>
+    <div className="headline">
```
Comment (tufoinnkuo10) [inline src/components/navbarHeading/navbar.js:7]: - [optional] you can make the nav link indicate which page a user is in by highlighting or changing the link color. nice work

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @keebot-png 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @keebot-png 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job working on the requirements provided by the previous reviewer ✅
- I liked that you generated random quotes 💯✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] The live demo link in your README file is broken, so I suggest to fix or remove it for now 💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**